target/ppc/insn32.decode

```diff
@@ -427,3 +430,7 @@ XXSPLTW         111100 ..... ---.. ..... 010100100 . .  @XX2
 ## VSX Vector Load Special Value Instruction
 
 LXVKQ           111100 ..... 11111 ..... 0101101000 .   @X_uim5
+
+## VSX Vector Test Least-Significant Bit by Byte Instruction
+
+XVTLSBB         111100 ... -- 00010 ..... 111011011 . -  @X_bf_xb
```

Comment:
Just noticed that this is an XX2-form instruction, so it should be `@XX2_bf_xb`

Reply:
Nice, fixed. Thanks